src/docfx/lib/HtmlUtility.cs

```diff
@@ -170,7 +170,7 @@ private static void AddLinkType(this HtmlNode html, string tag, string attribute
                 }
                 if (href[0] == '/' || href[0] == '\\')
                 {
-                    node.SetAttributeValue(attribute, AddLocaleIfMissing(HrefToLower(href), locale));
+                    node.SetAttributeValue(attribute, AddLocaleIfMissing(shouldToLowerCase ? HrefToLower(href) : href, locale));
```

Comment:
the `ToLower` was for the purpose of v2 backward compact, can you examine why some of the URL's are not converted to lower case in v2?

Reply:
we will leave this issue, add a TODO

---
In reply to: [212531773](https://github.com/dotnet/docfx/pull/3274#discussion_r212531773) [](ancestors = 212531773)